api/scanner/media_encoding/executable_worker/executable_worker.go

```diff
@@ -7,52 +7,68 @@ import (
 	"os"
 	"os/exec"
 	"strings"
+	"text/template"
+	"bytes"
+
 
 	"github.com/photoview/photoview/api/utils"
 	"github.com/pkg/errors"
 	"gopkg.in/vansante/go-ffprobe.v2"
 )
 
 func InitializeExecutableWorkers() {
-	DarktableCli = newDarktableWorker()
+	CustomRawCli = newCustomRawWorker()
 	FfmpegCli = newFfmpegWorker()
 }
 
-var DarktableCli *DarktableWorker = nil
+var CustomRawCli *CustomRawWorker = nil
 var FfmpegCli *FfmpegWorker = nil
 
 type ExecutableWorker interface {
 	Path() string
 }
 
-type DarktableWorker struct {
+type FfmpegWorker struct {
 	path string
 }
 
-type FfmpegWorker struct {
+type CustomRawWorker struct {
 	path string
+	args string
 }
 
-func newDarktableWorker() *DarktableWorker {
+type RawArgs struct {
+    InputPath string
+    OutputPath string
+	Quality int
+}
+
+
+func newCustomRawWorker() *CustomRawWorker {
 	if utils.EnvDisableRawProcessing.GetBool() {
-		log.Printf("Executable worker disabled (%s=1): darktable\n", utils.EnvDisableRawProcessing.GetName())
+		log.Printf("Executable worker disabled (%s=1) \n", utils.EnvDisableRawProcessing.GetName())
```

Comment:
Reviewing on mobile, but pretty sure darktavle should instead be replaced with raw processor so its clear on the logs which executable worker its referencing.

Reply:
Here is another environment variable that controls whether RAW format conversion is enabled. I don't think the value in another environment variable should be printed. Because if the user specifies this environment variable, another environment variable is likely to be null.